source/agora/common/TransactionPool.d

```diff
@@ -43,9 +43,8 @@ public class TransactionPool
 
     ***************************************************************************/
 
-    public static void initialize ()
+    shared static this()
     {
-        .shutdown();
```

Comment:
There is something which causes logging configuration to fail if shutdown is not called first. I'm not sure what it is yet. I'll add an integration test later to confirm it.

Reply:
First by compared to what? Nothing in calling order has changed.